### ISSUE TITLE
Release 1.32.0

### ISSUE THREAD

**Issue by aaemnnosttv** [google/site-kit-wp#3303]
---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Release is prepared, tested, and deployed to wordpress.org.

## Implementation Brief

* N/A

## QA Brief

* N/A

## Changelog entry

* N/A

**Comment by ivankruchkoff** [comment]
lgtm

**Comment by johnPhillips** [comment]
Looks good!

**Comment by wpdarren** [comment]
@aaemnnosttv on a new site, when I connect `PageSpeed Insights` from the dashboard this error message appears in the console `Google Site Kit API Error method:GET datapoint:pagespeed type:modules identifier:pagespeed-insights error:"You are probably offline."` and I am not offline. The module has connected successfully. 

I have not seen that message before. 

![image](https://user-images.githubusercontent.com/73545194/117304354-90485c00-ae75-11eb-9fb8-e55660e4698a.png)

**Comment by wpdarren** [comment]
@aaemnnosttv this could be my jurassic.ninja site but I am noticing `Error: The response is not a valid JSON response.` when trying to correct modules. It's very inconsistent, sometimes it's when I'm connecting Adsense, sometimes Analytics. 

Usually, when I refresh the page the error goes away.  Thought I would mention since it has happened a few times. 

![image](https://user-images.githubusercontent.com/73545194/117308231-41042a80-ae79-11eb-8321-bb4a351f37cd.png)

![image](https://user-images.githubusercontent.com/73545194/117308257-46fa0b80-ae79-11eb-9f7c-3b0ad41b46e6.png)

**Comment by aaemnnosttv** [comment]
@wpdarren those look like intermittent request failures which are not uncommon on jurassic.ninja (unfortunately?). The important thing is that those do not happen consistently.

**Comment by wpdarren** [comment]
@aaemnnosttv yes, it would just seem to be on jurrassic.ninja but [this is issue above](https://github.com/google/site-kit-wp/issues/3303#issuecomment-833515980 ) is consistently on my live site on the Site Kit dashboard when connecting page speed. Could you take a look and see if you can recreate it?

**Comment by Hazlitte** [comment]
Looks good to me ✅

**Comment by aaemnnosttv** [comment]
@wpdarren I didn't see any console errors when activating PageSpeed or getting the report on the dashboard, however this jump link no longer works on the widget based dashboard ⚠️ :
![image](https://user-images.githubusercontent.com/1621608/117335288-ab31c500-aea3-11eb-879e-bd066cdeadf5.png)

**Comment by eugene-manuilov** [comment]
Looks good to me.
